lib/errors/src/error.rs

```diff
@@ -9,6 +9,7 @@ pub enum Error {
     Encoding,
 
     // common
+    InvalidClusterData,
```

Comment:
Will error-num change affect spore-sdk?
@ShookLyngs

Reply:
Yes it will, because the contracts need to be redeployed.

Reply:
slightly changes will influent code hash of contracts, it's a little inconvenient for a minor update, what do you think of we let SDK fetch contract info from a version service?

Reply:
Personally, I believe that hardcoded code hashes can assist in identifying errors returned by RPC.
Maybe this can be seen as a "left hand or right hand" problem, you cannot have both.

Reply:
make sense, it leads to that we have to be cautious with our version upgrade, it's better to combine to a upgrade package